src/assets/styles/styles.scss

```diff
@@ -4,7 +4,7 @@
 // Colors
 $primary-color: #5603ad;
 $secondary-color: #ffffff;
-$third-primary: #b3e9c7;
+$third-primary: #f8f9fa;
```

Comment:
(suggestion; non-blocking): We should consider naming these variables the primary color, accent color, and tertiary color. Here's a link that may help: https://m3.material.io/styles/color/roles